moveit_ros/moveit_servo/include/moveit_servo/utilities.h

```diff
@@ -88,4 +89,23 @@ bool applyJointUpdate(rclcpp::Clock& clock, const double publish_period, const E
                       const sensor_msgs::msg::JointState& previous_joint_state,
                       sensor_msgs::msg::JointState& next_joint_state,
                       pluginlib::UniquePtr<online_signal_smoothing::SmoothingBaseClass>& smoother);
+
+/** \brief Converts a twist command from the command frame to the MoveGroup planning frame of the robot
```

Comment:
NitPick: Why MoveGroup frame? I think that name is only used in the context of the MoveGroup interface. Please correct me if I am wrong here. @AndyZe

Reply:
Maybe the function should be called `commandToPlanningFrame` :thinking:

Reply:
What about  just  `toPlanningFrame` ?

Reply:
I believe this is the root planning frame of MoveIt, so it's what OMPL and moveit_cpp would use, too. Not just move_group.

I kinda like `transformTwistToPlanningFrame()`